File: java_backup/java.jimut/new tution/factorial.java
import java.io.*;
class factorial
{
public static void main (String args[]) throws IOException
{
InputStreamReader ab = new InputStreamReader(System.in);
BufferedReader cd = new BufferedReader(ab);
int n,i,j;
double f=1;
System.out.println("ENTER THE VALUE WHICH YOU WANT TO MAKE FACTORIAL OF : ");
n = Integer.parseInt(cd.readLine());
{
for(i=1;i<=n;i++)
{
f = f * i;
}
System.out.println("FACTORIAL : "+f);
}
}
}

File: 1st_and_2nd_Sem_c_backup/BIN/C Prog/factorial.c
#include<stdio.h>

main()
{
	int i,x,p=1;
	
	printf("Enter The no. To Which Factorial is to be applied :- ");
	scanf("%d",&x);
	
	for(i=1;x>=i;x--)
	
	p=p*x;
	printf("the factorial is %d ",p);
	
}
